source/backend/tests/unit/dal/Repositories/PropertyRepositoryTest.cs

```diff
@@ -521,6 +522,47 @@ public void Update_Property_Success()
             updatedProperty.Description.Should().Be("test");
             updatedProperty.Pid.Should().Be(200);
         }
+        [Fact]
```

Comment:
Please just add one scenario where the existing property before update has Is_Retired as null. Since this will be the most likely scenario.

Reply:
updated